pandas/core/common.py

```diff
@@ -1596,7 +1596,7 @@ def is_timedelta64_dtype(arr_or_dtype):
 
 
 def is_timedelta64_ns_dtype(arr_or_dtype):
-    tipo = _get_dtype_type(arr_or_dtype)
+    tipo = _get_dtype(arr_or_dtype)
```

Comment:
This protects if we pass a dtype that numpy can't handle e.g. ``datetime64[ns, tz]``
so prob not necessary (as we would want it to blow up in that case as its obviously an error), for the datetimes its really a convenience thing